odyssey/client/src/Profiles/MyProfile.svelte

```diff
@@ -189,6 +227,26 @@
 			</div>
 
 		{/each}
+		{#each surveys as survey}
+			
+			<div class='post-box'>
+
+				<h3>{survey.text}</h3>
+
+				<img class="post-image" src={"/images/" + user.username + "/" + survey.image_path}>
+
+				{#if survey.is_open == true}
```

Comment:
User should be able to see the current results of the survey, not only the Close button

Reply:
Checked changes. Well done ❤️

Reply:
you can remove == true - https://svelte.dev/tutorial/if-blocks =))

Reply:
The issue is addressed in [this pull request](https://github.com/ogi02/Odyssey/pull/43). Thanks for bringing it up 😀